packages/admin-ui/src/__mock-backend__/data/dnps/types.ts

```diff
@@ -7,6 +7,8 @@ import {
   PackageContainer
 } from "../../../common";
 
+export { PortProtocol } from "../../../common";
```

Comment:
Merge duplicated imports in a single statement

Reply:
one is an import, the other one an export